src/Scene/Scene.js

```diff
@@ -215,18 +190,14 @@ Scene.prototype.scene3D = function () {
  *
  * @param node {[object Object]}
  */
-Scene.prototype.add = function (node, nodeProcess) {
-    if (node instanceof Globe) {
-        this.map = node;
-        nodeProcess = nodeProcess || new NodeProcess(this.currentCamera(), node.ellipsoid);
-        // this.quadTreeRequest(node.tiles, nodeProcess);
-    }
+Scene.prototype.add = function (layer, updater) {
+    this.layers.push({ layer, updater });
+    this.gfxEngine.add3DScene(layer.getMesh());
+};
 
-    this.layers.push({
-        node,
-        process: nodeProcess,
-    });
-    this.gfxEngine.add3DScene(node.getMesh());
+Scene.prototype.setMap = function (layer, updater) {
```

Comment:
I'll be removing this.map. See: https://github.com/iTowns/itowns2/pull/213#discussion_r91311375